learn/configuration/instance_options.mdx

```diff
@@ -136,8 +136,11 @@ Specifying the `config_file_path` option within the configuration file will thro
 ### Configuration file path
 
 **Environment variable**: `MEILI_CONFIG_FILE_PATH`
+
 **CLI option**: `--config-file-path`
+
 **Default**: `./config.toml`
+
 **Expected value**: a filepath
```

Comment:
Using `<br/>` like this will keep all sentences in the same paragraph (no margin at the end of each) but having one sentence for line.
```suggestion
**Environment variable**: `MEILI_CONFIG_FILE_PATH`<br/>
**CLI option**: `--config-file-path`<br/>
**Default**: `./config.toml`<br/>
**Expected value**: a filepath
```

Reply:
Got it.
We'll use `<br />` tags then. Thanks!